gips/0080-optimistic-ebo.md

```diff
@@ -0,0 +1,237 @@
+
+# **High-Level Description**
+
+## Optimistic EBO Solution Considerations
+
+The goal of this solution is to decentralize the EBO while keeping the structure open for future modifications.
+
+Key improvements:
+
+- **Optimistic operators:** Implementing an optimistic oracle will allow EBO operations to be open and permissionless, improving decentralization, redundancy, and scalability.
+- **Timestamp comparison:** Instead of querying each RPC when detecting a new epoch, a delay is introduced to compare timestamps, reducing time sensitivity and improving the precision of fork selection.
+- **Escalating bond mechanism**: The modularity of the optimistic oracle allows for an escalating bond dispute system, reducing Arbitrator intervention and enabling lower initial stakes without compromising security.
+    - The Arbitrator multisig will act as the ultimate source of truth in the case of disputes in this initial phase, with potential future improvements.
+
+### Optimistic Design
+
+![image.png](../assets//gip-0080/Optimistic%20EBO.png)
+
+### **Optimistic operators**
+
+The optimistic approach allows off-chain agents to propose updates to the Optimistic EBO, which other participants can dispute by staking tokens or using alternative resolution mechanisms to align incentives and ensure accuracy.
+
+For this, we will use [Prophet](https://github.com/defi-wonderland/prophet-modules/tree/dev), an optimistic, modular, and public-good oracle that abstracts the complexity of the proposal and dispute process while offering the flexibility to customize its functionality as needed. For more details on Prophet, refer to the documentation [here](https://docs.prophet.tech/content/modules/index.html).
+~~~~
+The process would work as follows: 
+
+- Any staked actor can propose an update for the EBO, which is published in a contract and remains open for a set period to allow for potential disputes.
+- Any staked actor can also challenge the proposed update. If a dispute arises, the initial proposal is discarded, and a new one can be submitted.
+- If a dispute occurs, the resolution mechanism is triggered, and the winner of the dispute will receive the counterparty’s bond as a reward.
+- If no disputes are raised within the designated time window, or if the dispute is resolved in favor of the proposer, the update is considered valid.
+
+Operators can update the block number of one chain at a time, allowing for ongoing disputes and resolutions for each updated chain. This improves the system's scalability.
+
+The [Detailed Specification](https://www.notion.so/DRAFT_I-GIP-0XYZ-Optimistic-EBO-74a0b9df7008464e9dc1215c46920213?pvs=21) section will provide further details on how contracts and off-chain agents interact.
```

Comment:
That specification is not, and likely will not be, publically visible? We should only link to information in the public domain?